intellst-api/src/DTO/UserDTO.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\DTO;
+
+use JMS\Serializer\Annotation as Serializer;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class UserDTO
```

Comment:
Where do you use this class? As I see, the request in controller isn't transformed in this DTO.